### PR TITLE
SSB4: Bugfixes and Balance Patches

### PR DESCRIPTION
**Pull request by xInstruct**
Turns out some of the "viable" moves from Data Corruption weren't viable at all, and were removed and added some other viable moves that were missed.

Also cut down on archas spam

### CODE REVIEW

File: data/mods/ssb/conditions.ts [hunk 1]
```diff
@@ -1248,13 +1248,16 @@ export const Conditions: {[k: string]: ModdedConditionData & {innateName?: strin
...
-			this.add(`c|${getName('PartMan')}|/me flees`);
+		onSwitchOut(source) {
+			this.add(`c|${getName('PartMan')}|Hi ${source.side.name}, I'm PartMan!`);
+			this.add(`c|${getName('PartMan')}|Hi PartMan, I'm PartMan!`);
+			this.add(`c|${getName('PartMan')}|Hi PartMan, I'm PartMan!`);
+			this.add(`c|${getName('Hydro')}|/log PartMan was muted by Hydro for 7 minutes. (flood)`);
 		},
 		onFaint() {
-			this.add(`c|${getName('PartMan')}|B-booli. >.<`);
+			this.add(`c|${getName('PartMan')}|${['B-booli. >.<', 'That hurt me in my meow-meow. ;-;', 'Remember to dab on iph', 'Excuse me what', 'RUDE', ':pout:'][this.random(6)]}`);
```
Comment (KrisXV) [inline data/mods/ssb/conditions.ts:1260]: > That hurt me in my meow-meow. ;-;

no

File: data/mods/ssb/moves.ts [hunk 2]
```diff
@@ -722,14 +702,14 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 				move.accuracy = true;
 				move.target = "self";
 				move.flags.protect = 0;
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts:704]:
```suggestion
				delete move.flags.protect;
```

**Comment by Zarel** [comment]
As usual, unless something is urgent, let's wait a weekish to batch changes in one PR.

**Comment by xInstruct** [comment]
Yh noted. Though on the 15th/16th-ish there will be winrates so I dont think anything should be merged before then.

**Comment by xInstruct** [comment]
This can be merged now because I don't want the conflict gremlins to sneak up behind me while QC preps the rest of the stuff.

**Pull request merged**